app/src/openxr/cpp/OpenXRHelpers.h

```diff
@@ -126,4 +127,18 @@ inline XrPosef MatrixToXrPose(const vrb::Matrix& aMatrix) {
     return result;
 }
 
+typedef std::array<XrHandJointLocationEXT, XR_HAND_JOINT_COUNT_EXT> HandJointsArray;
+inline bool IsHandJointPositionValid(const enum XrHandJointEXT aJoint, const HandJointsArray& handJoints) {
```

Comment:
Nit: wouldn't an empty line improves readability ?

Reply:
Maybe, I can add it if you want.